CMakeLists.txt

```diff
@@ -37,5 +40,18 @@ add_fatelf_executable(fatelf-verify)
 add_fatelf_executable(fatelf-split)
 add_fatelf_executable(fatelf-validate)
 
+set(CMAKE_REQUIRED_DEFINITIONS -D_GNU_SOURCE)
+check_symbol_exists(memfd_create sys/mman.h MEMFD)
+if(MEMFD)
+    add_fatelf_executable(fatelf-exec)
+
+    find_program(BINFMT systemd-binfmt PATHS /usr/lib/systemd)
```

Comment:
I'm on the fence as to whether `/usr/lib` ought to be replaced with `${CMAKE_INSTALL_FULL_LIBDIR}`.